src/frontend/src/pages/DesignReviewCommon.tsx

```diff
@@ -0,0 +1,122 @@
+import { Box } from '@mui/system';
+import { ReactElement } from 'react';
+import { User } from 'shared';
+import WarningIcon from '@mui/icons-material/Warning';
+import BuildIcon from '@mui/icons-material/Build';
+import ComputerIcon from '@mui/icons-material/Computer';
+import ElectricalServicesIcon from '@mui/icons-material/ElectricalServices';
+
+export interface DRCModalProps {
+  open: boolean;
+  title: string;
+  onHide: () => void;
+  onSubmit?: () => void;
+  usersToAvailabilities: Map<User, number[]>;
+  iconData: Map<number, string>;
```

Comment:
this should be named existingMeetingData in both components, since that what the iconData actually represents